stories/core_components/HeatMap.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { screenshotWrap } from '@gooddata/test-storybook';
+
+import { HeatMap } from '../../src/index';
+import { onErrorHandler } from '../mocks';
+import {
+    ATTRIBUTE_1,
+    ATTRIBUTE_2,
+    MEASURE_1,
+    ATTRIBUTE_1_WITH_ALIAS
+} from '../data/componentProps';
+
+const wrapperStyle = { width: 800, height: 400 };
+
+storiesOf('Core components/HeatMap', module)
+    .add('metric row column', () => (
```

Comment:
run Storybook tests and add new screenshots

Reply:
done, screenshot added